backend/connection.go

```diff
@@ -312,65 +312,65 @@ type Frame struct {
 }
 
 func (c *Connection) watchFile(action Action) {
```

Comment:
Ah I see, you are vendoring the 0.4.0 version, that's why. Okay, let's leave it out for now.